google/resource_storage_bucket_object.go

```diff
@@ -108,6 +112,21 @@ func objectGetId(object *storage.Object) string {
 	return object.Bucket + "-" + object.Name
 }
 
+func createSourceState(src interface{}) string {
+
+	source := src.(string)
+	data, err := ioutil.ReadFile(source)
+	if err != nil {
+		return source
+	}
+
+	h := md5.New()
+	h.Write(data)
+	md5 := base64.StdEncoding.EncodeToString(h.Sum(nil))
+
+	return source + "." + md5
```

Comment:
Rather than using a "." as the separator which is commonly found in filenames, how about something like ":"?

Reply:
Done